exercise/templates/exercise/staff/results.html

```diff
@@ -10,45 +11,185 @@
 <li class="active">{% trans "All results" %}</li>
 {% endblock %}
 
+{{% block scripts %}
+{{ block.super }}
+<!--Library to handle multiselection on select tags-->
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/css/bootstrap-multiselect.css">
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/js/bootstrap-multiselect.min.js"></script>
+
+<!--Required libraries for exporting table data to xlsx, csv or txt.-->
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/css/tableexport.min.css">
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.14.3/xlsx.core.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/js/tableexport.min.js"></script>
+
+<!--Custom for loading animation (loading_animation.css) and other extra customizations (results_staff.css)
+    results_staff.js is the main script for this page. -->
+<link rel="stylesheet" href="{% static 'exercise/css/loading_animation.css' %}" />
+<link rel="stylesheet" href="{% static 'exercise/css/results_staff.css' %}" />
+<script src="{% static 'exercise/results_staff.js' %}"
+        allExercisesUrl="/api/v2/exercises/"
+        exercisesUrl="/api/v2/courses/{{course.id}}/exercises"
+        studentsUrl="/api/v2/courses/{{course.id}}/students"
+        pointsUrl="/api/v2/courses/{{course.id}}/points/"
+        usersUrl="/api/v2/users/"
+        usertagsUrl="/api/v2/courses/{{course.id}}/usertags"
+        taggingsUrl="/api/v2/courses/{{course.id}}/taggings"
+        externalLabel="{{ external_user_label|safe }}"
```

Comment:
Are these needed still?

Reply:
If they are needed, they should be data attributes

Reply:
These were not needed